### DISCUSSION TITLE
How long does an inactive session last?

### DISCUSSION THREAD

**Discussion by FrancoisLeb** [InseeFrLab/onyxia#873] [General]
Hello,
I'd like to know how long a session lasts when it's inactive. I'm using Jupyter Notebook on Onyxia, and I'd like to know how long it will be deleted for inactivity?
Thanks in advance

**Comment by fcomte** [comment]
Hello ,

Onyxia does not expire services
But some instance of Onyxia may delete your service with their own policy.

For SSPCloud, we may delete any services that is older than 7 days.
For SSPCloud, if you reserve gpu and don't use it during 2 hours , your service will be stopped to release the GPU.

**Comment by FrancoisLeb** [comment]
Thanks for this information.
Do you know if it's possible to extend the inactivity time for some users?
I want to extend an current session for some users from 7 days, to 15 days.
May be we can talk in private

**Reply by fcomte** [reply]
It is not possible.
In sspcloud token to S3 storage are only valid for 7 days 
15 days of "interactive" seem a bit a long

**Comment by FrancoisLeb** [comment]
And, to reactivate it for another 7 days, we just need to open the session ? Or we need to do something else ?

**Reply by garronej** [reply]
Consider automating the setup process for your services to make recovery simpler in case of shutdowns. You could start by automating the cloning of your Git repository. For more details, check out the setup guide:
https://docs.onyxia.sh/user-doc/setting-up-your-dev-environment-in-onyxia
